README.md

```diff
@@ -10,17 +10,17 @@ See LICENSE.
 First, read how
 [dartdoc structures links](https://github.com/dart-lang/dartdoc/blob/master/README.md#link-structure).
 
-The api.dart.dev server prepends some structure to the links from dartdoc.
+The api.dart.dev server appends some structure to the links from dartdoc.
```

Comment:
Neither prepends or appends is precisely right here, I think it said prepends because "beta" is being replaced by "3.2.0-123.1.beta", for example.  I would say "expands the links from dartdoc", or "redirects".

Reply:
Expands is a good word. I'll use it.